apps/src/p5lab/P5LabView.jsx

```diff
@@ -76,6 +81,8 @@ class P5LabView extends React.Component {
       responsive: isResponsive,
       pin_bottom: !hideSource && pinWorkspaceToBottom
     });
+    const libraryManifest = spriteLab ? spriteCostumeLibrary : animationLibrary;
+    let categories = spriteLab ? CostumeCategories : AnimationCategories;
```

Comment:
nit: move the logic for these lines into helper functions (e.g., `this.getLibraryManifest()` and `this.getCategories()`) just in case we need to change this logic in the future